### ISSUE TITLE
Outline still has reference to book, not publication

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_publications#80]
At https://localgov.ddev.site/node/2/outline

![Image](https://github.com/localgovdrupal/localgov_publications/assets/326588/6b18bc47-595d-48bc-8620-956d6ab074f1)

**Comment by graham-web** [comment]
Changing the book reorder links to the publications equivalents would be useful - an editor we support was trying to reorder by manually adjusting weights on this page  (which causes a white screen here...!) as they hadn't found the main publications reorder interface.

**Comment by msayoung** [comment]
Yes, I agree, if we could update the link to the /admin/structure/publications here that would be useful. 

By default the editors aren't seeing the Publications option in the menu, although they do have the permission to use the view. Although Rupert's comment here: https://github.com/localgovdrupal/localgov_publications/issues/148#issuecomment-2468107202 may also solve this.
